src/server/helpers.py

```diff
@@ -67,7 +69,8 @@ def render_error_template(error, status_code):
         elif os.path.isfile(TEMPLATES_DIR + '/%s/%s/error.html' % (DEFAULT_LANGUAGE.lang_code, DEFAULT_YEAR)):
             lang = DEFAULT_LANGUAGE.lang_code
             year = DEFAULT_YEAR
-    return render_template('%s/2019/error.html' % lang, lang=lang, year=year, error=error), status_code
+    return render_template('%s/2019/error.html' % lang, lang=lang, year=year, error=error,
+                           get_file_date_info=get_file_date_info), status_code
```

Comment:
Rather than adding the function kwarg to each template, I think this qualifies for the [global Jinja variables list](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/server/__init__.py#L55):

Alternatively, what about baking the published/modified dates directly into the overridden [`render_template`](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/server/helpers.py#L9) function, similar to how we make supported years/languages globally available? That way there are no functions to call and no arguments to manage, eg templates can simply render `{{ last_modified_date }}`.

Reply:
Good idea! Fixed.

We should do this for config also.